src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -88,11 +91,41 @@ public class Checker extends AbstractAutomaticBean implements MessageDispatcher,
     /** The audit event filters. */
     private final FilterSet filters = new FilterSet();
 
+    /** Message used by destroy method. */
+    private final LocalizedMessage destroyMsg = new LocalizedMessage(
+            Definitions.CHECKSTYLE_BUNDLE, getClass(),
+                    "Checker.destroy");
+
     /** Message used by finishLocalSetup method. */
     private final LocalizedMessage finishLocalSetupMsg = new LocalizedMessage(
             Definitions.CHECKSTYLE_BUNDLE, getClass(),
                     "Checker.finishLocalSetup");
 
+    /** Message used by processFiles method to show Error. */
+    private final LocalizedMessage processFilesErrorMsg = new LocalizedMessage(
+            Definitions.CHECKSTYLE_BUNDLE, getClass(),
+                    "Checker.processFilesError");
```

Comment:
That's a lot of repetitive code.

1. Could you try to single out common part into a method to end up with something like this: `private final LocalizedMessage processFilesErrorMsg = localizedMessage("Checker.processFilesError");`.
2. Do we actually need those fields? Why can't we use inplace values? Considering, they are used once in rare cases when exception is thrown, I think we a good to have them closer to the usage.